projects/components/src/multi-select/multi-select.component.test.ts

```diff
@@ -91,8 +93,8 @@ describe('Multi Select Component', () => {
     });
 
     spectator.tick();
-    const selectedElements = spectator.queryAll('input:checked', { root: true });
-    expect(selectedElements.length).toBe(2);
+    const checkboxElements = spectator.queryAll('ht-checkbox', { root: true });
+    expect(checkboxElements.length).toBe(6);
```

Comment:
How did this change to 6? You are only migrating to checkbox component, so the count shouldn't change

Reply:
The content is changing because in the first case are testing if the input is checked and in the second one it has all checkboxElements. In this case, there are 6 checkbox elements and two of them are checked.

Reply:
I'll go back to the last PR, renaming the variables to have better clarity on the elements I'm manipulating.

Reply:
Sure, I will find time for it. Should be easy to fix